### ISSUE TITLE
Ride music style blanco when music from RTD isn't available

### ISSUE THREAD

**Issue by karst** [OpenRCT2/OpenRCT2#21138]
### Operating System

Win 10, 64-Bit

### OpenRCT2 build

OpenRCT2, v0.4.7-2-gf6840cb (f6840cb on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2, RollerCoaster Tycoon Classic

### Area(s) with this issue?

This issue has to do with park objects and/or asset packs

### Describe the issue

Current behaviour:
When building a ride that doesn't have it's default ride music style (RMS) available/selected in the object selector the RMS in the ride's window will be blanco.

Expected behaviour: 
A random available RMS will be selected, if available, or display a text that no RMS is available if none are selected.

### Steps to reproduce

1. Start "Build your own Six Flags Park"
2. Open Object selector > Advanced > RMS
3. Deselect Dodgems beat style
4. Build a dodgems
5. Check RMS window of the ride

### Attachments

![image](https://github.com/OpenRCT2/OpenRCT2/assets/1516893/b2e0dd24-c28d-43e6-8df5-87ff332ea165)

**Comment by 733737** [comment]
https://github.com/OpenRCT2/OpenRCT2/issues/20745  shows different behavior. Unless the difference is before checking the play music bix versus after checking it

**Comment by karst** [comment]
They're related, but it's a different bug

**Comment by 733737** [comment]
perhaps we could simply hide the music tab when there is no music to be selected for this ride type. That should fix both issues.

**Comment by karst** [comment]
Although I like that idea for the current behaviour. We would like to add a tab where you can select any RMS without cheats. Lets say a very toned down version of the object selector.

**Comment by 733737** [comment]
to clarify would this tab include even custom music objects not currently enabled in the park?

**Comment by karst** [comment]
Correct, that would essentially finish the replacement of CUSTOM1.WAV and CUSTOM2.WAV
